### PR TITLE
Update Keycloak operator example with correct hostname syntax

### LINKED ISSUE #26818: bug in operator example yaml
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

operator

### Describe the bug

Area
operator

Describe the bug
I was reading through the discourse forum today, and noticed this post https://keycloak.discourse.group/t/is-the-keycloak-operator-still-maintained/20350 which prompted me to look at https://operatorhub.io/operator/keycloak-operator and notice that the example has an incorrectly formatted hostname section:

```yaml
hostname:
  hostname: example.org
```

Version
22.x

Expected behavior
ability to install and configure the operator given the examples provided

Actual behavior
unable to install the operator due to invalid example

How to Reproduce?
Use the given example structure:
apiVersion: k8s.keycloak.org/v2alpha1
kind: Keycloak
metadata:
  name: example-keycloak
  labels:
    app: sso
spec:
  instances: 1
  hostname: example.org
  tlsSecret: my-tls-secret
:(
Anything else?
https://keycloak.discourse.group/t/is-the-keycloak-operator-still-maintained/20350

### Version

22

### Expected behavior

correctly formatted example for deploying Keycloak from the Keycloak operator

### Actual behavior

error on deploying Keycloak operator due to incorrectly formatted yaml

### How to Reproduce?

example in description

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by BadgerOps**
Closes #26818

What: _Another_ Update for the Keycloak Operator documentation as per issue https://github.com/keycloak/keycloak/issues/26281

Why: To ensure new users to the Keycloak Operator have a good example to follow!

I just realized that the hostname stanza was also incorrect, so submitting a new PR (correctly DCO'd!) to fix that too. Let me know if you'd rather me create a new issue for this.

This is a pretty simple change, just ensuring the example [here](https://operatorhub.io/operator/keycloak-operator) is correct.

### CODE REVIEW

**Comment by ahus1** [comment]
The change was missing a `,` so it wasn't valid JSON any more. I've added one more commit.

@BadgerOps - please create a new issue describing the expected vs. actual example for this and link it here.

@shawkins - please review the change.

**Comment by BadgerOps** [comment]
Hi @ahus1 :doh: ! I was going for first time success and failed once again 😬  Thanks for that catch with the comma - that file is super hard to parse as a human, haha.

Here is a new issue with the reported bug https://github.com/keycloak/keycloak/issues/26818 

thanks again!

-BadgerOps

**Review by vmuzikar — APPROVED** [review]
@BadgerOps Thank you for spotting this! LGTM.
